runtime/near-vm/lib/types/src/types.rs

```diff
@@ -577,11 +577,11 @@ pub struct InstanceConfig {
     pub gas_counter: *mut FastGasCounter,
     default_gas_counter: Option<Rc<UnsafeCell<FastGasCounter>>>,
     /// Stack limit, in 8-byte slots.
-    pub stack_limit: i32,
+    pub stack_limit: u32,
 }
 
 // Default stack limit, in bytes.
-const DEFAULT_STACK_LIMIT: i32 = 1024 * 1024;
+const DEFAULT_STACK_LIMIT: u32 = 256 * 1024;
```

Comment:
Hm, so this is only relevant to wast tests. I would argue that we shouldn't have any default at all and callers should specify the limits directly. With `wast` specifying whatever is appropriate to it, and near-vm-runner specifying its own limits too.

Reply:
Done :)